socorro/processor/mozilla_transform_rules.py

```diff
@@ -868,101 +871,75 @@ def _action(self, raw_crash, raw_dumps, processed_crash, proc_meta):
 
 
 class OSPrettyVersionRule(Rule):
-    required_config = Namespace()
-    required_config.add_option(
-        'database_class',
-        doc="the class of the database",
-        default='socorro.external.postgresql.connection_context.'
-                'ConnectionContext',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
-    required_config.add_option(
-        'transaction_executor_class',
-        default="socorro.database.transaction_executor."
-                "TransactionExecutorWithInfiniteBackoff",
-        doc='a class that will manage transactions',
-        from_string_converter=str_to_python_object,
-        reference_value_from='resource.postgresql',
-    )
+    '''This rule attempts to extract the most useful, singular, human
+    understandable field for operating system version. This should always be
+    attempted.
+    '''
+
+    WINDOWS_VERSIONS = {
```

Comment:
I lifted this from static data, and used a supersearch query faceted on different version fields to verify that this is _probably_ the right data. That said, it's worth checking the prod database to make sure I got the data right (I no longer have access to make this kind of check myself, sorry)